docs/installation.md

```diff
@@ -10,7 +10,7 @@ kubectl apply -n argo-rollouts -f https://github.com/argoproj/argo-rollouts/rele
 This will create a new namespace, `argo-rollouts`, where Argo Rollouts controller will run.
 
 !!! tip
-    If you are using another namspace name, please update `install.yaml` clusterrolebinding's serviceaccount namespace name.
+    If you are using another namespace name, please update `install.yaml` clusterrolebinding's serviceaccount namespace name.
```

Comment:
```suggestion
    If you are using another namespace, update `install.yaml` ClusterRoleBinding's ServiceAccount namespace to the one you are using.
```

`name` seems redundant here. If we are at it, could we also just drop the `name` here?